ci-operator/config/stackrox/collector/stackrox-collector-master.yaml

```diff
@@ -358,11 +358,21 @@ tests:
       VM_TYPE: fedora-coreos
     workflow: stackrox-collector-integration-tests
 - as: flatcar-integration-tests
-  postsubmit: true
```

Comment:
I believe this postsubmit should be left in, right? Or were we missing the presubmit for flatcar?

Reply:
yeah there were basically two post submit jobs for flatcar (the other being the more usual `merge-flatcar-integration-tests`) so removing this make it run on PRs (admittedly only when the all-integration-tests label is provided)